### ISSUE TITLE
Long int in source is different that JSONView shows

### ISSUE THREAD

**Issue by noose** [gildas-lormeau/JSONVue#26]
JSONView

``` javascript
{
category_id: 127,
id: 1187874342096751900,
[...]
}
```

Source:

``` javascript
{"category_id":127,"id":1187874342096751953,[....]}
```

Can you fix it? :-)

**Comment by gildas-lormeau** [comment]
Indeed, the extension uses `JSON.parse` so it cannot be fixed.

**Comment by gildas-lormeau** [comment]
This issue will be fixed in the version 0.1.17 by enabling the option "Support 64-bit signed integers".
